apps/beeai-server/src/beeai_server/api/routes/embeddings.py

```diff
@@ -25,18 +24,18 @@ class EmbeddingsRequest(pydantic.BaseModel):
 
     model: str
     input: list[str] | str
-    encoding_format: Literal["float"]
+    encoding_format: str | None = None
```

Comment:
You probably want this

```suggestion
    encoding_format: Literal["float"] = "float"
```

Watsonx does not support the base64 format and I did not want to bother polyfilling it since afaik it's not used in our agents, I'd keep the arg like this in the meantime so that the error message is clear when somebody tries to use base64

Reply:
voyage supports only "base64" here or `null` (not float):
https://docs.voyageai.com/reference/embeddings-api

so it needs to be at least nullable, but I wouldn't make opinions here, the error message should be clear from the embedding backend - maybe you know about it and you designed your agents that way

Reply:
ok makes sense, added the remap